components/automate-chef-io/content/docs/quickstart.md

```diff
@@ -14,46 +14,88 @@ toc = true
 Hello and welcome to Chef Automate 2! This document guides you through the initial installation and trial activation.
 
 The easiest way to start exploring Chef Automate is to install it on a local VM.
-We recommend using [Vagrant][] to create your test environment.
+We recommend using [Vagrant](https://www.vagrantup.com/) to create your test environment.
 Use this Vagrantfile to provision a VM. Chef recommends a minimum of 4GB RAM.
 
+## Add an entry to /etc/hosts
+
+The example Vagrantfile sets the VM hostname to `chef-automate.test`. The installer uses this hostname as the FQDN that Chef Automate listens on.
```

Comment:
"The example Vagrantfile sets the VM hostname..." -> "Throughout this example, we will use `chef-automate.test` as the hostname for the VM," or similar.

Reply:
Docs avoids hedges like "Throughout this example". We should consider introducing the acronym since it is in the quickstart `uses this hostname as the fully qualified domain name (FQDN)`.